ts/utils/__tests__/internalLink.test.ts

```diff
@@ -21,6 +28,15 @@ describe("getInternalRoute", () => {
         IO_INTERNAL_LINK_PREFIX + validRoute + "?param1=value1&param2=value2",
         some({
           routeName: validRoute,
+          navigationAction: {
+            payload: {
+              name: "MAIN",
```

Comment:
Can we use constants here and in the other parts of this file?

Reply:
Removed in 4f0c96984120de6802df1a45e18d42786fd523dd